README.md

```diff
@@ -2328,6 +2328,64 @@ In order to achieve greater coverage and encourage more people to contribute to
                 </a>
             </td>
         </tr>
+        <tr>
+            <td><a href="https://en.wikipedia.org/wiki/Circular_buffer">Ring Buffer (Circular Buffer)</a></td>
```

Comment:
I understand that this data structure has many names but I think its important to standardize, so I would suggest using only "Ring Buffer":
```suggestion
            <td><a href="https://en.wikipedia.org/wiki/Circular_buffer">Ring Buffer</a></td>
```